pkgs/top-level/python-packages.nix

```diff
@@ -9454,6 +9456,8 @@ self: super: with self; {
 
   nunavut = callPackage ../development/python-modules/nunavut { };
 
+  nupd = callPackage ../development/python-modules/nupd { };
```

Comment:
```suggestion
  nixpkgs-updaters-library = callPackage ../development/python-modules/nixpkgs-updaters-library { };
```
same here?

Reply:
I would like to keep the name short of this package, I changed nbdb to nonbloat-db though

Reply:
The project name is like that https://github.com/PerchunPak/nixpkgs-updaters-library/blob/main/pyproject.toml#L2 and we use that everywhere, for the attr, the pname and the directory. If we would allow custom nuances in the naming, it would be impossible to find anything. Also since nupd is not taken on pypi.org, there could easily come up a package in the future that clashes with this name.